apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -4,8 +4,30 @@
       <oc-table-cell shrink :colspan="firstColumn ? 2 : 1"></oc-table-cell>
       <oc-table-cell colspan="2">
         <div class="uk-text-meta uk-flex uk-flex-middle">
-          <oc-icon name="repeat" class="uk-preserve-width" />
-          <span class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</span>
+          <oc-icon name="repeat" class="uk-preserve-width oc-icon-xsmall" />
+          <a :id="$_resharerToggleId" :aria-label="$gettext('Show resharer details')" class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</a>
+          <oc-drop
+            :dropId="$_resharerToggleId + '-drop'"
+            :toggle="'#' + $_resharerToggleId"
+            mode="click"
+            :options="{pos:'bottom', delayHide: 0}"
+            class="uk-width-medium"
+            ref="menu"
+            closeOnClick
+          >
+            <h4><translate>Reshared by:</translate></h4>
```

Comment:
Can be made shorter with:

`<h4 v-translate>Reshared by:</h4>`

or

`<translate tag="h4">Reshared by:</translate>`